scripts/utils/constructProposal.ts

```diff
@@ -11,13 +11,13 @@ import format from 'string-template';
  * Uses the data in `proposals/description/${proposalName}.json` for the commands
  * Uses the text in `proposals/description/${proposalName}.txt` for the description
  */
-export default async function constructProposal(proposalName: string, logging = false) {
+export default async function constructProposal(proposalName: string, logging = false, contracts = undefined, contractAddresses = undefined) {
```

Comment:
this change was necessary to include contracts deployed at runtime

Reply:
fine by me, though we have an existing pattern of keeping the logging flag at the end of the function arg list, we should try to keep it that way if we can

Reply:
fixed